bifrost/proxy/pages/onRenderClient.tsx

```diff
@@ -62,13 +73,7 @@ export default async function onRenderClient(
       copyElementAttributes(document.body, bodyEl);
       // render body with react
       render(bodyEl.innerHTML);
+      cachePageContext();
     });
   }
-
-  // cache page context will save it and return it to us during restoration visits
-  Turbolinks._vpsCachePageContext({
```

Comment:
If we navigate from page A to B, we need to save page B's pageContext after caching page A. caching page A happens async as part of the `_vpsOnRenderClient` flow.

Thus, we need to move this caching into the render flow or it happens too early. 

Spec: https://github.com/Alignable/bifrost/blob/1eb3ffdf7e02e911f9629f5b735eb05b65b15dd6/tests/e2e/specs/e2e.spec.ts#L470